src/exams/exams.controller.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Body,
+  Controller,
+  Param,
+  ParseIntPipe,
+  Post,
+  UseGuards,
+  UseInterceptors,
+} from "@nestjs/common";
+
+import { Roles } from "@/auth/decorators/roles.decorator";
+import { JwtAuthGuard } from "@/auth/guards/jwt-auth.guard";
+import { RolesGuard } from "@/auth/guards/roles.guard";
+import { EUserRole } from "@/common/enums/roles.enum";
+import { ResponseTransformInterceptor } from "@/common/interceptors/response-transform.interceptor";
+
+import { CreateExamDto } from "./exams.dtos";
+import { ExamsService } from "./exams.service";
+
+@UseInterceptors(ResponseTransformInterceptor)
+@UseGuards(JwtAuthGuard)
+@Controller("classrooms")
+export class ExamsController {
+  constructor(private readonly examsService: ExamsService) {}
+
+  @Post(":id/exams")
```

Comment:
```suggestion
  @Post(":classroomId/exams")
```


[NIT] - Best to be precise with the id names